lib/trust_is_risk.js

```diff
@@ -0,0 +1,202 @@
+//      
+var bcoin = require('bcoin');
+var assert = require('assert');
+var Address = bcoin.primitives.Address;
+
+                      // base58 bitcoin address
+                    
+                
+             
+                 
+                 
+                                                               
+ 
+                               
+                     
+
+class TrustIsRisk {
+                       
+  
+                 
+                        
+                            
+      
+   
+
+               
+                                 
+   
+
+  constructor(node                 ) {
+    this.node = node;
+    this.directTrust = {};
+    this.TXToTrust = {};
+
+    this.node.on('tx', this.addTX.bind(this));
+  }
+
+  getDirectTrust(from         , to         )          {
+    if (!this.directTrust.hasOwnProperty(from)) return 0;
+    if (!this.directTrust[from].hasOwnProperty(to)) return 0;
```

Comment:
Use [in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in)